package/jailer.sh

```diff
@@ -54,6 +55,7 @@ fi
 cp -r -l /opt/drivers/management-state/bin /opt/jail/$NAME/var/lib/rancher/management-state
 
 # Hard link rancher-machine into the jail
+
```

Comment:
Major nitpick, but we don't need this new empty line.

Reply:
Fixed in latest patchset.